backend/internal/storage/postgres/schema/media.go

```diff
@@ -0,0 +1,71 @@
+package schema
+
+import (
+	"context"
+	"platnm/internal/models"
+	"github.com/jackc/pgx/v5/pgxpool"
+)
+
+type MediaRepository struct {
+	*pgxpool.Pool
+}
+
+func (r *MediaRepository) GetMediaByName(ctx context.Context, name string) ([]models.Media, error) {
+
+	// TODO: TWEAK THE SIMILARITY SCORE MAX HERE
```

Comment:
remove the todo after updating the query !

Reply:
Done! Added searching via the LIKE operator, so if the keyword is in the title it will also return in search.